### ISSUE TITLE
Error when disabling Use Imperial Calculation for Fuel Economy

### ISSUE THREAD

**Issue by thehijacker** [hargata/lubelog#12]
When I change _"Use Imperial Calculation for Fuel Economy Calculations(MPG) This Will Also Change Units to Miles and Gallons"_ I get and error:

`An error occurred, please try again later.`

Docker gives and error:

```
07. 01. 2024 09:57:06 fail: CarCareTracker.Controllers.HomeController[0]
07. 01. 2024 09:57:06      Error on saving config file.
07. 01. 2024 09:57:06      System.IO.FileNotFoundException: Could not find file '/App/config/userConfig.json'.
07. 01. 2024 09:57:06      File name: '/App/config/userConfig.json'
07. 01. 2024 09:57:06         at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
07. 01. 2024 09:57:06          at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Boolean failForSymlink, Boolean& wasSymlink, Func`4 createOpenException)
07. 01. 2024 09:57:06
         at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
07. 01. 2024 09:57:06         at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
07. 01. 2024 09:57:06         at System.IO.File.ReadAllText(String path, Encoding encoding)
07. 01. 2024 09:57:06         at CarCareTracker.Controllers.HomeController.WriteToSettings(UserConfig userConfig) in /App/Controllers/HomeController.cs:line 58
```

**Comment by thehijacker** [comment]
Solved it by creating file userConfig.json under /config folder with content:

```
{
}
```

**Comment by hargata** [comment]
this bug is fixed in the #14
